test/libyul/yulOptimizerTests/redundantStoreEliminator/remove_before_revert.yul

```diff
@@ -0,0 +1,19 @@
+{
+    let c := calldataload(0)
```

Comment:
This is already working nicely. The component currently has no idea about constants, so it cannot see that the revert does not actually read from memory.

Reply:
Now it can.